### ISSUE TITLE
ERR_CACHE_MISS error when trying to get back to object page

### ISSUE THREAD

**Issue by smarttgit** [lsst-uk/lasair#42]
After running a stream, getting a table, selecting an object and going to the object page. 

Then I can't always go back to the table, I often (but not always) get 

Confirm Form Resubmission
This web page requires data that you entered earlier in order to be properly displayed. You can send this data again, but by doing so you will repeat any action this page previously performed.
Press the reload button to resubmit the data needed to load the page.
ERR_CACHE_MISS

Any chance of fixing this, or ... when a user clicks on a link, then a new tab pops up, so that the original table is still available in a tab

**Comment by genghisken** [comment]
Yes - let's try and fix this.  It drives me crazy because Lasair has to re-run the query when you resubmit the form.  If the query takes 4.9 mins each time, this can get really annoying.  But it's also inconsistent.  Sometimes you can go back no probs!

I'm not sure there's a simple fix (probably internal to Django form submission mechanics), so we might just have to cache the results.

**Comment by RoyWilliams** [comment]
I have made each object link open a new tab in your browser, thus keeping the tab with the original query response. Hope this helps.
